src/build.jl

```diff
@@ -86,7 +86,7 @@ end
 
 
 function demand(H::Consumer)
-    D = demands(model(H))
+    D = demand_dict(model(H))
```

Comment:
Changed `demands` function returns a flat list of all the demands. This matches the `productions` and `sectors` functions. However, it used to return the `demands` field from the model, which was `Dict{ScalarConsumer, ScalarDemand}`. The `demand_dict` function returns exactly this.